addons/l10n_cn_small_business/data/account.account.template.csv

```diff
@@ -1,77 +1,69 @@
+"l10n_cn_1413","低值易耗品","1413","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1421","消耗性生物资产","1421","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1501","持有至到期投资","1501","account.data_account_type_non_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1511","长期股权投资","1511","account.data_account_type_non_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1601","固定资产","1601","account.data_account_type_fixed_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1602","累计折旧","1602","account.data_account_type_depreciation","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1604","在建工程","1604","account.data_account_type_non_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1605","工程物资","1605","account.data_account_type_non_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1606","固定资产情况","1606","account.data_account_type_depreciation","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1621","生产性生物资产","1621","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1622","生产性生物资产累计折旧","1622","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1701","无形资产","1701","account.data_account_type_non_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1702","累计摊销","1702","account.data_account_type_depreciation","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1801","长期待摊销费用","1801","account.data_account_type_depreciation","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1901","待处理财产损溢","1901","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_2001","短期借款","2001","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2201","应付票据","2201","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2202","应付账款","2202","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2203","预收账款","2203","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2211","应付职工薪酬","2211","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2221","应交税费","2221","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
```

Comment:
Is this account need  reconcile ?

Reply:
@GoodERPJeff Normally, all payable accounts need to allow the reconciliation. Why do you think they shouldn't be in reconciled=true ?